use/httpserver.rst

```diff
@@ -24,20 +24,22 @@ The HTTP comes with a couple of additional variables:
 Using the server
 ================
 
-At the root of the server, you will find a list of the plugins that have a Web
-interface, and a link to them. Each plugin has its own subdirectory(ies).
+At the root of the server, you will find a list of the plugins that
+have a Web interface, and a link to them. Each plugin has one or more
```

Comment:
Should web be spelled with small letter? It seems to have been written with capital letter before, but I am not sure if that is correct.